rest/model/distro.go

```diff
@@ -8,14 +8,16 @@ import (
 // APIDistro is the model to be returned by the API whenever distros are fetched.
 // EVG-1717 will implement the remainder of the distro model.
 type APIDistro struct {
-	Id APIString `json:"_id"`
+	Id           APIString `json:"_id"`
```

Comment:
I've never liked that this is _id, maybe we could change it?

Reply:
I recognize that this is outside of the scope of the pull request.

Reply:
Created EVG-2164 for that.